design/quarterly-release.md

```diff
@@ -0,0 +1,49 @@
+# Change Contour releases to quarterly
+
+Status: Accepted
+
+## Abstract
+Move Contour and Contour Operator to a longer release cadence to cushion for release planning, align Contour releases with Contour Operator releases, and align with downstream VMware TKG releases, and improve feature planning. 
+
+Please see [#3634](https://github.com/projectcontour/contour/issues/3634) for more discussion.
+
+## Current issues with monthly release
+* Very tough for users to keep up with Contour monthly releases. It’s rare for production deployments to always chase the latest version and more likely to upgrade every 6 months or longer. As part of these changes, we will be investigating how possible enabling jump upgrades is for Contour.
+* Release engineering is taxing and we have to spend some time preparing for this every month. The tradeoff here is that we will need to do more patch releases, and that quarterly releases will take more effort to put together than monthly ones, simply because they'll be bigger.
+* Since we release the Contour Operator 1:1 with Contour, the Operator release adds to the complexity of release engineering.  An issue with a Contour release will thus affect Contour Operator pushing it out and in turn eat into the next release cycle
+
+## Proposal
+
+*  Move from existing model of only supporting the latest release to supporting N-2 model, i.e. 3 minor releases (denoted by W, X, Y) at any given time.  For example, with the release of Contour v1.22, we are responsible for maintaining v1.22, v1.21, and v1.20.
+*  The version version to be covered by this schedule will be Contour v1.20. Please see the "Rollout Process" section below for details.
+*  The following will be reasons we will consider issuing a patch release:
+   * CVE reported for Contour or an upstream component of Contour like Envoy
+   * Critical bugs 
+   * Feature enhancement requests with enough community support. This will require an exceptional circumstance and a lot of community support.
+* If we issue a patch release for the latest minor version, we will also backport it to all supported versions.
+* We will also make at least one Release Candidate (RC) build available before each *minor* release, to enable Contour's downstream consumers to test and validate before Contour releases. This RC build will be released at least two weeks before a minor release.
```

Comment:
We'll need a convention here on how to create images against this, etc. I assume this is just a tag off of :main?

Reply:
That probably makes sense, we could also start the release branch early and tag RC builds off there. That would make the eventual release less of a big change.